### ISSUE TITLE
Sessionbased webhook-url

### ISSUE THREAD

**Issue by neucrk** [chrishubert/whatsapp-api#295]
Is it possible (and if so, how) to define session-based webhook-url via an api-call?

Thank you for this project! ;)

**Comment by avoylenko** [comment]
There is only one option to set the env variable for each configured session

**Comment by lazmo88** [comment]
@avoylenko could you please clarify, each session can have one webhook configuration in the env file? What would be the best way handle dynamic webhook rules for any number of sessions?

**Comment by avoylenko** [comment]
Right, each wwebjs session uses just one webhook configuration. You might need an extra reserve proxy to handle your scenario.

**Comment by kaua-alves-queiros** [comment]
Very good! I had already suggested doing this because I also use NSwag to generate the SDK

**Comment by devitbash** [comment]
I use two webhooks, you must only configure on your .env ie.:

If you have the sessionId defined as DEMO, the environment variable must be DEMO_WEBHOOK_URL but define too a default webhook (it could be the default webhook)
